drivers/clock/sunxi-ccu.c

```diff
@@ -27,21 +24,13 @@ sunxi_ccu_disable(struct device *clockdev, struct device *dev)
 	uint8_t reset;
 
 	if ((reset = CCU_GET_RESET(dev->clock))) {
-		mmio_clearbits32(clockdev->regs + CCU_RESET_BASE +
-		                 BITMAP_WORDOFF(reset),
-		                 BIT(BITMAP_BIT(reset)));
-		if (mmio_read32(clockdev->regs + CCU_RESET_BASE +
-		                BITMAP_WORDOFF(reset)) &
-		    BIT(BITMAP_BIT(reset)))
+		bitmap_clear(clockdev->regs + CCU_RESET_BASE, reset);
```

Comment:
Some comments about why you are checking these conditions would be nice to have here - otherwise one has to reference the manual.